src/components/sync-sites-modal-selector.tsx

```diff
@@ -49,39 +52,49 @@ export function SyncSitesModalSelector( {
 			onRequestClose={ onRequestClose }
 			title={ __( 'Connect a WordPress.com site' ) }
 		>
-			<SearchSites searchQuery={ searchQuery } setSearchQuery={ setSearchQuery } />
-			<div className="h-[calc(84vh-232px)]">
-				{ isLoading && (
-					<div className="flex justify-center items-center h-full">{ __( 'Loading sites…' ) }</div>
-				) }
+			<div className="relative">
+				<SearchSites searchQuery={ searchQuery } setSearchQuery={ setSearchQuery } />
+				<div className="h-[calc(84vh-232px)]">
+					{ isLoading && (
+						<div className="flex justify-center items-center h-full">
+							{ __( 'Loading sites…' ) }
+						</div>
+					) }
 
-				{ ! isLoading && isEmpty && (
-					<div className="flex justify-center items-center h-full">
-						{ searchQuery
-							? sprintf( __( 'No sites found for "%s"' ), searchQuery )
-							: __( 'No sites found' ) }
-					</div>
-				) }
+					{ ! isLoading && isEmpty && (
+						<div className="flex justify-center items-center h-full">
+							{ searchQuery
+								? sprintf( __( 'No sites found for "%s"' ), searchQuery )
+								: __( 'No sites found' ) }
+						</div>
+					) }
+
+					{ ! isLoading && ! isEmpty && (
+						<ListSites
+							syncSites={ filteredSites }
+							selectedSiteId={ selectedSiteId }
+							onSelectSite={ setSelectedSiteId }
+						/>
+					) }
+				</div>
+				<Footer
+					onRequestClose={ onRequestClose }
+					onConnect={ () => {
+						if ( ! selectedSiteId ) {
+							return;
+						}
+						onConnect( selectedSiteId );
+						onRequestClose();
+					} }
+					disabled={ ! selectedSiteId }
+				/>
 
-				{ ! isLoading && ! isEmpty && (
-					<ListSites
-						syncSites={ filteredSites }
-						selectedSiteId={ selectedSiteId }
-						onSelectSite={ setSelectedSiteId }
-					/>
+				{ isOffline && (
+					<div className="absolute inset-0 bg-white/80 z-10 flex items-center justify-center">
```

Comment:
Not sure if this needs more or less transparency. I tried with something like `bg-white/90` and it seems like it is not transparent enough to me. Happy to adjust this though

Reply:
This looks good to me. 👍